### PR TITLE
Add Attila Zsolt Piros to committers

### PR DESCRIPTION
**Pull request by attilapiros**

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ navigation:
 |Kazuaki Ishizaki|IBM|
 |Xingbo Jiang|Databricks|
 |Holden Karau|Apple|
+|Attila Zsolt Piros|Cloudera|
```
Comment (HeartSaVioR) [inline committers.md]: Same comment here. (I agree this is quite confusing.)

Actually there's a convention in the list if I understand correctly (I don't know where this is documented) - the list is alphabetically sorted by "last name" (not first name).

Could you please check the sequence a bit and please update the same if the list follows the pattern?

**Review by HeartSaVioR — APPROVED** [review]
Congrats!

File: site/committers.html [hunk 2]
```diff
@@ -338,6 +338,10 @@ <h2>Current Committers</h2>
       <td>Holden Karau</td>
       <td>Apple</td>
     </tr>
+    <tr>
+      <td>Attila Zsolt Piros</td>
+      <td>Cloudera</td>
+    </tr>
```
Comment (HyukjinKwon) [inline site/committers.html]: Don't forget to update htmls too man :-)

**Review by HyukjinKwon — APPROVED** [review]
Congrats!

File: site/committers.html (hunk 2 above)
Comment (attilapiros) [reply site/committers.html]: I am testing my check action ;)

**Comment by HeartSaVioR** [comment]
Go test with your new ability to push! :)

**Comment by attilapiros** [comment]
Sure I will :) I am just checking the inside of the `merge_pr.py` whether I can improve it.

TL;DR
As I have two factor authentication for github the https based remotes are deprecated for me I know this as I got a mail:

```
 You recently used a password to access the repository at apache/spark-website with git using git/2.21.1 (Apple Git-122.3).

Basic authentication using a password to Git is deprecated and will soon no longer work. Visit https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/ for more information around suggested workarounds and removal dates.

Thanks,
The GitHub Team
```

Visiting the link you will found:
> If you have two-factor authentication enabled for your account, you are already required to use token- or SSH-based authentication.

On the other hand using SSH is not working in `merge_pr.py`:

My remote:
```
apache	git@github.com:apache/spark-website.git (fetch)
apache	git@github.com:apache/spark-website.git (push)
```

My error:
```
Traceback (most recent call last):
  File "./merge_pr.py", line 287, in <module>
    main()
  File "./merge_pr.py", line 278, in main
    merge_pr(pr_num, target_ref, title, body, pr_repo_desc)
  File "./merge_pr.py", line 111, in merge_pr
    run_cmd("git fetch %s pull/%s/head:%s" % (PR_REMOTE_NAME, pr_num, pr_branch_name))
  File "./merge_pr.py", line 86, in run_cmd
    return subprocess.check_output(cmd.split(" ")).decode('utf-8')
  File "/Users/attilazsoltpiros/.pyenv/versions/3.8.1/lib/python3.8/subprocess.py", line 411, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/Users/attilazsoltpiros/.pyenv/versions/3.8.1/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['git', 'fetch', 'apache-github', 'pull/328/head:PR_TOOL_MERGE_PR_328']' returned non-zero exit status 128.
```

So I will fix this first :) Stay tuned!

**Comment by attilapiros** [comment]
Ok so what I see we have to add both `apache` and `apache-github` as remote:
```
apache	git@github.com:apache/spark-website.git (fetch)
apache	git@github.com:apache/spark-website.git (push)
apache-github	git@github.com:apache/spark-website.git (fetch)
apache-github	git@github.com:apache/spark-website.git (push)
```


In the code:
```
# Remote name which points to the Github site
PR_REMOTE_NAME = os.environ.get("PR_REMOTE_NAME", "apache-github")
# Remote name which points to Apache git
PUSH_REMOTE_NAME = os.environ.get("PUSH_REMOTE_NAME", "apache")
```

Probably there is historical reason why this two is separated. 

But I still have some problems with ssh based auth:
```
git push apache PR_TOOL_MERGE_PR_328_ASF-SITE:asf-site
Warning: Permanently added 'github.com,140.82.121.3' (RSA) to the list of known hosts.
ERROR: Permission to apache/spark-website.git denied to attilapiros.
fatal: Could not read from remote repository.
```

This must be just a missing step from the documentation. 
I think I found all the hiccups so the next committer with two factor auth will be a lucky guy :)

**Review by gaborgsomogyi — APPROVED** [review]
Congrats!

**Comment by attilapiros** [comment]
GitBox synch is running this was the missing step

**Comment by attilapiros** [comment]
Here are my PR mentioning GitBox, documenting those two remotes and show why they are very useful and using SSH based remotes:
https://github.com/apache/spark-website/pull/329

**Comment by zhengruifeng** [comment]
Congrats!
